tools/pipelines/templates/include-vars.yml

```diff
@@ -101,5 +101,5 @@ variables:
 # Indicates if tests should be run with code coverage analysis.
 # We don't do it for PRs to make those runs faster.
```

Comment:
Code-coverage analysis will increase the PR pipeline duration?  If yes, Pragya may be interested in knowing the impact

Reply:
Yes, since we will need to calculate the code coverage too while running the tests. Although I don't expect it to be much. It is some cost that we would like to pay as the current code coverage is not much. Enforcing this would slowly increase the coverage.

Reply:
@pragya91 FYI. I am just testing for now. Once I figure out, I can check the time it increases for the builds.